hurumap_tz/static/css/hurumap.css

```diff
@@ -0,0 +1,237 @@
+  background-position: center;
+  min-height: 300px;
+  padding: 150px 0; }
+  #homepage .hero select, #homepage .hero a {
+    max-width: 100%;
+    width: 320px;
+    height: auto;
+    display: inline-block;
+    padding: 13px 20px 12px;
+    background-color: #1c9d95;
+    text-transform: uppercase;
+    text-align: left;
+    color: #fff;
+    border: none;
+    font-size: 16px;
+    font-family: "Poppins", sans-serif;
+    border-radius: 5px;
+    box-shadow: 0 0 20px #000;
+    cursor: pointer; }
+    #homepage .hero select i, #homepage .hero a i {
+      float: right;
+      margin-top: 2px; }
+    #homepage .hero select:hover, #homepage .hero a:hover {
+      background-color: #084a49; }
+  #homepage .hero #locate-me {
+    padding-right: 13px; }
+
+.showcase {
+  padding: 50px 0; }
+  .showcase .column-third .img {
+    height: 200px;
+    width: 100%;
+    max-width: 300px;
+    margin: 10px auto 20px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center; }
+  .showcase.partners img {
+    max-width: 250px;
+    max-height: 75px;
+    margin: 10px; }
+    .showcase.partners img:hover {
+      opacity: 0.7; }
+
+.btn {
+  text-decoration: none; }
+  .btn.btn-lg {
+    font-size: 1.2em;
+    padding: 10px 20px 12px; }
+  .btn.bg-ocean-green:hover {
+    background-color: #084a49;
+    color: #fff; }
+  .btn.bg-bottle-green:hover {
+    background-color: #1c9d95;
+    color: #fff; }
+
+.select2-container .select2-choice div {
+  background-color: #1c9d95 !important;
+  border: none !important;
+  background-image: url("/static/img/arrow-down-white.png") !important;
+  background-size: 24px !important;
+  background-repeat: no-repeat !important;
+  background-position: left center !important;
+  width: 34px !important; }
+  .select2-container .select2-choice div b {
+    display: none !important; }
+
+.select2-container .select2-choice:hover, .select2-container .select2-choice:focus, .select2-container .select2-choice:hover div, .select2-container .select2-choice:focus div {
+  background-color: #084a49 !important; }
+
+.select2-container .select2-default {
+  color: #fff !important; }
+
+.select2-drop {
+  padding: 10px 5px; }
+  .select2-drop .select2-results {
+    font-size: 1.1em !important; }
+    .select2-drop .select2-results .select2-result-label {
+      padding: 5px 8px 6px; }
+    .select2-drop .select2-results .select2-highlighted {
+      background-color: #084a49; }
+
+#page .container-narrow, #page .container-narrowest {
+  padding-top: 50px;
+  padding-bottom: 50px; }
+
+.leaflet-label {
+  color: #fff !important;
+  background-color: #1c9d95 !important;
+  border-color: #1c9d95 !important;
+  border-radius: 0 !important;
+  font-weight: 200 !important;
+  font-family: "Poppins", sans-serif !important;
+  padding: 3px 6px 1px !important; }
+
+.how-it-works img {
+  max-width: 70px; }
+
+/*# sourceMappingURL=../../../../../../../static/css/hurumap.css.map */
```

Comment:
This file probably should be deleted and is being caused because of the `STATIC_ROOT` being changed in `settings.py`. If there are no CSS changes, the `hurumap/static/css` folder should be deleted.

Reply:
Sure, the `STATIC_ROOT` setting was getting the `css` folder regenerated. I've removed the  setting and the `css` folder